ibis/backends/tests/conftest.py

```diff
@@ -285,11 +285,13 @@ def temp_view(con) -> str:
 @pytest.fixture(scope='session')
 def current_data_db(con, backend) -> str:
     """Return current database name."""
-    if not hasattr(con, 'current_database'):
+    try:
+        current_database = con.current_database
```

Comment:
You can `return con.current_database` instead of creating a variable and returning later.